tools/circt-lec/circt-lec.cpp

```diff
@@ -122,6 +122,65 @@ static cl::opt<OutputFormat> outputFormat(
 // Tool implementation
 //===----------------------------------------------------------------------===//
 
+// Move all operations in `src` to `dest`. Rename all symbols in `src` to avoid
+// conflict.
+static FailureOr<StringAttr> mergeModules(ModuleOp dest, ModuleOp src,
+                                          StringAttr name) {
+
+  SymbolTable destTable(dest), srcTable(src);
+  StringAttr renamed = {};
```

Comment:
Super-nit: Could we call this something like `newName`? To me `renamed` implies a boolean indicating whether it's been renamed.